docs/webapp.adoc

```diff
@@ -427,7 +427,7 @@ Just like before, we get react/react-dom from a CDN, provide a pinch of styling
         <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
 
         <!--1-->
-        <script data-th-src="${portal.serviceUrl({'_service=react4xp-client'})}" src=""></script>
+        <script data-th-src="${portal.serviceUrl({'_service=react4xp'})}/client.5678abcd.js" src=""></script>
```

Comment:
This won't work - it's not rendered output HTML, but a template file, so therefore `portal.serviceUrl`. But that would mean the developer needs to knows in advance the hash of the asset.

Reply:
Feel free to fix the docs, I've just tried to change the things that no longer exits. Like the react4xp-client service.

Reply:
@sigdestad The webapp "flow" propbably doesn't work anymore. The service it used has been removed. I think we should discuss what to do about that...

Reply:
Fixed